python/test/unit/language/test_core.py

```diff
@@ -4732,6 +4732,34 @@ def kernel():
     assert "reshape" in str(exc_info.value)
 
 
+def test_tma_load_block_shape_err():
+
+    @triton.jit
+    def kernel(ptr):
+        desc = tl._experimental_make_tensor_descriptor(ptr, [128, 128], [128, 1], [1, 32])
+        desc.load([0, 0])
+
+    input = torch.empty((128, 128), dtype=torch.int32, device='cuda')
+    with pytest.raises(triton.CompilationError) as e:
+        kernel[(1, )](input)
+
+    assert "tensor descriptor block shape must have at least 8 rows" in str(e.value.__cause__)
```

Comment:
Does using `raises` to match not work because it's only in the exception cause?

Reply:
Yeah. The original AssertionError is placed in the cause